server/schedule/region_splitter.go

```diff
@@ -119,3 +136,57 @@ func (r *RegionSplitter) groupKeysByRegion(keys [][]byte) (map[uint64][][]byte,
 	}
 	return groupKeys, unProcessedKeys
 }
+
+func (r *RegionSplitter) checkRegionValid(region *core.RegionInfo) bool {
+	if r.cluster.IsRegionHot(region) {
+		return false
+	}
+	if !opt.IsRegionReplicated(r.cluster, region) {
+		r.cluster.AddSuspectRegions(region.GetID())
+		return false
+	}
+	if region.GetLeader() == nil {
+		return false
+	}
+	return true
+}
+
+type splitRegionsHandler struct {
+	cluster opt.Cluster
+	oc      *OperatorController
+}
+
+func (h *splitRegionsHandler) SplitRegionByKeys(region *core.RegionInfo, splitKeys [][]byte) error {
+	op := operator.CreateSplitRegionOperator("region-splitter", region, 0, pdpb.CheckPolicy_USEKEY, splitKeys)
+	if ok := h.oc.AddOperator(op); !ok {
+		log.Info("add region split operator failed", zap.Uint64("regionID", region.GetID()))
```

Comment:
```suggestion
		log.Info("add region split operator failed", zap.Uint64("region-id", region.GetID()))
```
BTW, should the level be `info`?

Reply:
updated.